canonical-websites/www.ubuntu.com
# Update openstack overview page

## Done

Update page as per [copy doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit?ts=5b0045b1#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/openstack>
- Make sure page matches the [copy doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit?ts=5b0045b1#) - please check off comments as you verify these changes


## Issue / Card

Fixes #3201
